addons/auth_signup/tests/test_reset_password.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests.common import HttpCase
+
+
+class TestResetPassword(HttpCase):
+
+    def test_reset_password(self):
+        """
+            Test that first signup link and password reset link are different
```

Comment:
Note that it does not explain what is the issue with "signup_email" being present in a reset password

Reply:
I updated the comment

        """
            Test that first signup link and password reset link are different to accomodate for the different behaviour
            on first signup if a password is already set user is redirected to login page when accessing that link again
            'signup_email' is used in the web controller (web_auth_reset_password) to detect this behaviour
        """